v0/template.json

```diff
@@ -56,7 +56,7 @@
             },
             "versionLabel" : {
               "type" : "string",
-              "pattern" : "^[0-9a-zA-Z][^\\s/&]{0,63}$"
+              "pattern" : "^[0-9a-zA-Z][^\\s/&]{0,127}$"
```

Comment:
@deepak-harness @archit-harness Should this be 127?

Reply:
Yes, i think it should be 127, please update for v1 as well